src/content-type-visibility/application/content-type-visibility-dismiss-notifications.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Yoast\WP\SEO\Content_Type_Visibility\Application;
+
+use WP_REST_Request;
+use WP_REST_Response;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast_Notification_Center;
+
+/**
+ * Handles dismissing notifications and "New" badges for new content types.
+ */
+class Content_Type_Visibility_Dismiss_Notifications {
+
+	/**
+	 * Holds the Options_Helper instance.
+	 *
+	 * @var Options_Helper
+	 */
+	private $options;
+
+	/**
+	 * Constructs Content_Type_Visibility_Dismiss_New_Route.
+	 *
+	 * @param Options_Helper $options The options helper.
+	 */
+	public function __construct( Options_Helper $options ) {
+		$this->options = $options;
+	}
+
+	/**
+	 * Dismisses an alert.
+	 *
+	 * @param WP_REST_Request $request The request. This request should have a key param set.
+	 *
+	 * @return WP_REST_Response The response.
+	 */
+	public function post_type_dismiss( $request ) {
+		$success = true;
+		$message = 'Post type is not new.';
+
+		$post_types_needs_review = $this->options->get( 'new_post_types', [] );
+
+		if ( \in_array( $request['postTypeName'], $post_types_needs_review, true ) ) {
```

Comment:
Not sure where to put this but I would use the request name post_type_name. that is more inline with other naming conventions

Reply:
If I will do that then in the JS side I'll get a warning for not following naming conventions... Maybe I need something neutral, one word, maybe `type`?